### PR TITLE
Adding FF

### PR DESCRIPTION
**Pull request by hectorpal**
In general, I'm following https://github.com/ronuchit/pddlgym_planners/blob/master/ff.py

- Download FF from https://fai.cs.uni-saarland.de/hoffmann/ff
- Provision for installing in Mac
- Added APT packages build-essential, flex, bison
- Clean up instalation and execution

### CODE REVIEW

File: planutils/packages/ff/uninstall [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+rm ./bin/ff
```
Comment (haz) [inline planutils/packages/ff/uninstall:3]: Shouldn't the `.gz` be removed too?

File: planutils/packages/ff/manifest.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+    "name": "Fast-Forward",
+    "description": "https://fai.cs.uni-saarland.de/hoffmann/ff.html",
+    "install-size": "1.1M",
+    "dependencies": [],
+    "endpoint": {
+        "services": {
+            "solve": {
```
Comment (haz) [inline planutils/packages/ff/manifest.json:8]: Should probably just use the template, like lama-first:
https://github.com/AI-Planning/planutils/blob/main/planutils/packages/lama-first/manifest.json

This is what it inserts: https://github.com/AI-Planning/planutils/blob/main/planutils/packages/TEMPLATE/SERVICE_TEMPLATE/planner.json

**Review by haz — CHANGES_REQUESTED** [review]

File: planutils/packages/ff/uninstall (hunk 1 above)
Comment (hectorpal) [reply planutils/packages/ff/uninstall:3]: gunzip deletes the `.gz` file

File: planutils/packages/ff/manifest.json (hunk 2 above)
Comment (hectorpal) [reply planutils/packages/ff/manifest.json:8]: Oh, I get it.

**Review by haz — APPROVED** [review]
Tried and tested, thanks!

**Pull request merged**